pagerduty/resource_pagerduty_automation_actions_action_test.go

```diff
@@ -154,6 +210,30 @@ resource "pagerduty_automation_actions_action" "foo" {
 `, actionName, actionName)
 }
 
+func testAccCheckPagerDutyAutomationActionsActionTypeProcessAutomationConfigUpdated(oldName, updatedName, updatedDescription, updatedActionClassification string) string {
```

Comment:
Hey @mrdubr could you please update this function parameters names to something like: `(runnerName, actionName, description, actionClassification string)`? This in order to be more descriptive and consistent with the rest of the updated configuration functions